.github/workflows/builds.yml

```diff
@@ -628,9 +628,19 @@ jobs:
         uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16
         with:
           name: dist
+
+      - name: Build and install nDPI
+        run: |
+          curl -OL https://github.com/ntop/nDPI/archive/refs/tags/4.12.tar.gz
```

Comment:
Can the version number be refactored with a variable so it's easier to maintain across version updates?